README.org

```diff
@@ -3,6 +3,9 @@
 Utilities and DSL for logging on top of [[https://github.com/ptaoussanis/timbre][Timbre]] with support for [[https://github.com/riemann/riemann][Riemann]], 
 [[https://www.elastic.co/de/logstash][Logstash]], and more.
 
+One /goal/ of this library is to funnel log messages coming from
+different libraries that talk to =slf4j= into =active-logger=.
+
```

Comment:
Currently =active-logger= supports hijacking log messages from Log4j, java.util.logging (JUL), and Apache Commons Logging (JCL).